ew/static/cosmetics.py

```diff
@@ -2178,8 +2178,8 @@
     EwCosmeticItem(
         id_cosmetic = "widebrimmedhat",
         str_name = "Wide Brimmed Hat",
-        str_desc = "This hat cost you so many hours, you might as well be the lord of this city! Better not let the patrons see you in it though... They keep asking you to... Crush them...",
-        str_onadorn = "You place the wide brimmed hat on your head, the edge flopping over one eye sexily. Oh yeah, lets get this ~~blood~~ bread.",
+        str_desc = "This hat cost you so many hours, you might as well be the lord of this city! Better not let the patrons see you in it though, They keep asking you to... kill them...",
```

Comment:
the original was more correct than your fix. glimmer made it suggestive idk why you need to remove this